daemon.c

```diff
@@ -112,19 +112,46 @@ void logmsg(int prio, const char *fmt, ...)
 }
 
 /*
- * return remote address from svc_req structure
+ * return remote address from svc_req structure in IPv6 format
+ * 
+ * IPv4 addresses are mapped to IPv6
+ *
+ * for AF_INET6, remote_buf is not used and ptr inside svc_req struct is returned
+ * for AF_INET, remote_buf is used and its poineter is returned
  */
-const struct in6_addr *get_remote(struct svc_req *rqstp)
+const struct in6_addr *get_remote(struct svc_req *rqstp, struct in6_addr *remote_buf) 
```

Comment:
Myslím, že tohle mělo být spíš static než const. Kdyžtak na to udělejte samostatný commit.

Reply:
Const je dobře - nechceme, aby někdo do adresy zapisoval, pokud není z remote_buf.

Static být nemůže, protože je využívaná v dalších souborech.